roles/pbx/tasks/asterisk.yml

```diff
@@ -55,6 +55,12 @@
 #     name: aptitude
 #     state: latest
 
+# https://github.com/iiab/iiab/issues/3489
+- name: Apply patch to {{ asterisk_src_dir }}/contrib/scripts/install_prereq for 3489
+  ansible.posix.patch:
+    src: install_prereq.diff
+    dest: "{{ asterisk_src_dir }}/contrib/scripts/install_prereq"
+
```

Comment:
A `when:` clause should be inserted here, so patch is applied in a targeted way.

Maybe something like below... ?

```
when: php_version is version('8.0', '>=') and pbx_php74_override is defined
```

Reply:
Not really needed, you will never get past main.yml with php 8+ and the patch does the right action with any workflow.